<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="ViewList_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>331</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="FirstAddLine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Код книги               |  Код читателя       |  код книги</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Название книги      | Ф.И.О читателя    |  Код читателя</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SecondAddLine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Автор книги            | Номер телефона   | Дата выдачи</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ThirdAddLine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата публикации   | Адресс читателя  | Срок возврата</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FourthAddLine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Язык локализации |                                 | дата возврата</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FifthAddLine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AddBook">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>добавить книгу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddUser">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>250</y>
      <width>106</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>добавить читателя</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddList">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>102</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>добавить в список</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BooksFromFile">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>добавить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>добавить информацию о книгах из файла</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UsersFormFile">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>добавить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>добавить информацию о читателях из файла</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ListFromFile">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>490</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>добавить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>добавить информацию о выданных книгах из файла</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>  код книги   |       код читателя      | код книги и читателя</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteBook">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>320</y>
      <width>84</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>удалить книгу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteUser">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>320</y>
      <width>110</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>удалить читателя</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteList">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>320</y>
      <width>110</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>удалить из списка</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DeleteLine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="FindBook">
    <property name="geometry">
     <rect>
      <x>366</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Найти книгу</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="BookFindAuthor">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>360</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="BookFindTitle">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>400</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>340</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Автор книги</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>362</x>
      <y>380</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Название книги</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>367</x>
      <y>460</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ф.И.О читателя</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FindUser">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>510</y>
      <width>100</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Найти читателя</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="UserFindName">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>480</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ViewBooks">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>340</y>
      <width>120</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>показать Книги</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ViewUsers">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>440</y>
      <width>120</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>показать читателей</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ViewList">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>390</y>
      <width>120</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>показать выданые</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ViewDelay">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>490</y>
      <width>120</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>показать должников</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SortBooks">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>сортировать книги</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SortUsers">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>520</y>
      <width>130</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>сортировать читателей</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
